stream-video-android-ui-compose/src/main/kotlin/io/getstream/video/android/compose/theme/base/StreamColors.kt

```diff
@@ -0,0 +1,115 @@
+/*
+ * Copyright (c) 2014-2024 Stream.io Inc. All rights reserved.
+ *
+ * Licensed under the Stream License;
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://github.com/GetStream/stream-video-android/blob/main/LICENSE
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.getstream.video.android.compose.theme.base
+
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.res.colorResource
+import io.getstream.video.android.ui.common.R
+
+public data class StreamColors(
+    val brandPrimary: Color,
+    val brandPrimaryLt: Color,
+    val brandPrimaryDk: Color,
+    val brandSecondary: Color,
+    val brandSecondaryTransparent: Color,
+    val brandCyan: Color,
+    val brandGreen: Color,
```

Comment:
I've noticed that we currently have duplicate APIs for theming components, which likely confuses our users regarding which one to utilize. Despite sharing the same name, we must consolidate these two duplicate APIs into a unified API to streamline the user experience.

![Screenshot 2024-03-07 at 9 02 05 AM](https://github.com/GetStream/stream-video-android/assets/24237865/9a901a24-d34e-4c36-bba3-42646ba959a5)